cmd/buildah/images.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	imagesFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "omit column headings",
+		},
+		cli.BoolFlag{
+			Name:  "qq",
```

Comment:
The --quiet flag is what we already for headings in "list", that'll need to be changed there to keep them consistent.  Updating.